crates/core/src/main.rs

```diff
@@ -43,16 +43,15 @@ pub extern "C" fn init() {
     }
 }
 
-#[export_name = "shopify_main"]
```

Comment:
I think I'm missing some context on this change

Reply:
Oh the addition context here is that this `main` function by default is assigned the `_start` name

Reply:
> Oh the addition context here is that this main function by default is assigned the _start name

Yeah that was the piece I was missing 🙂. Thank you for elaborating!